SayTheirNames/Source/Controller/Home/Person/Views/PersonNewsCollectionViewCell.swift

```diff
@@ -7,87 +7,110 @@
+    private var news: News?
+    
     static var reuseIdentifier: String {
         return "\(Self.self)"
     }
     
     lazy var newsImageView: UIImageView = {
         let imageView = UIImageView()
-        imageView.image = UIImage(asset: STNAsset.Image.mediaImage1)
+        imageView.image = UIImage(asset: STNAsset.Image.placeholder)
         imageView.contentMode = .scaleAspectFill
         imageView.clipsToBounds = true
         return imageView
     }()
     
-    lazy var sourceContainer: UIView = {
-        let view = UIView()
-        view.backgroundColor = UIColor(asset: STNAsset.Color.yellow)
-        view.clipsToBounds = true
-        view.addSubview(sourceImageView)
-        return view
-    }()
-    
-    lazy var sourceImageView: UIImageView = {
-        let imageView = UIImageView()
-        imageView.image = UIImage(asset: STNAsset.Image.newsBbc)
-        imageView.contentMode = .scaleAspectFill
-        return imageView
-    }()
-    
-    lazy var sourceInfoLabel: UILabel = {
+    lazy var newsTitleLabel: UILabel = {
         let label = UILabel()
-        label.text = "Live updates: Another night of fire and fury as.."
+        label.text = ""
         label.numberOfLines = 3
         label.font = UIFont.STN.sectionHeader
-        label.adjustsFontSizeToFitWidth = true
         label.minimumScaleFactor = 0.5
         label.textColor = UIColor(asset: STNAsset.Color.detailLabel)
         return label
     }()
     
+    lazy var loadingIndicator: UIActivityIndicatorView = {
+        let view = SDWebImageActivityIndicator.gray.indicatorView
+        view.hidesWhenStopped = true
+        return view
+    }()
+    
     override init(frame: CGRect) {
         super.init(frame: frame)
         backgroundColor = UIColor(asset: STNAsset.Color.background)
         setupLayout()
+        
+        // Start loading indicator
+        loadingIndicator.startAnimating()
     }
     
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
     
+    func configure(with news: News) {
+        self.news = news
+        
+        if let url = URL(string: news.url) {
+            self.metadata.fetchLinkInformation(from: url) { [weak self] result in
+                // This is a weird check, but it prevents a reusedCell from loading incorrect information.
+                // TODO: Long term, move this loading into Combine/Futures and move it back into a controller.
+                if let oldNews = self?.news, news == oldNews {
+                    
+                    switch result {
+                    case .success(let link):
+                        DispatchQueue.main.async {
+                            self?.newsImageView.image = link.image
+                            self?.newsTitleLabel.text = link.title
+                            self?.loadingIndicator.stopAnimating()
+                        }
+                    case .failure(let error):
+                        print(url)
```

Comment:
Does the `print` statement still need to be there since there's a log statement right after ?